js/interviewQs/knives.js

```diff
@@ -30,21 +30,59 @@ const knives = {
 const food = {
   allfood : [ {
         name:'Butter',
-        psiSlice: .15
+        psiSlice: .15,
+        spoilAge: 45,
+        currentAge: 3 // days
     
       },
      {
       name:'Meat',
-      psiSlice: .8
+      psiSlice: .8,
+      spoilAge: 7,
+      currentAge: 1
       },
       {
         name:'Onions',
-        psiSlice: .56
+        psiSlice: .56,
+        spoilAge: 10,
+        currentAge: 2 
     
       }
     ]
 }
 
+class Knife {
+  constructor(name,handle,thickness,psi,location){
+    this.name = name
+    this.handle = handle
+    this.thickness = thickness
+    this.psi = psi
+    this.location = location
+  }
+  cut(psi){
```

Comment:
Change the parameters from psi to accept an input of any misc. food obj insert

Function Call Example: ButcherKnife.cut('Onions').....